src/webpack/webpack.config.js

```diff
@@ -0,0 +1,141 @@
+const Autoprefixer = require('autoprefixer');
+const PostCSSAssetsPlugin = require('postcss-assets-webpack-plugin');
+const PostCSSCustomProperties = require('postcss-custom-properties');
+const path = require('path');
+const rtl = require('postcss-rtl');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CleanPlugin = require('clean-webpack-plugin');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const aggregateTranslations = require('../../scripts/aggregate-translations/aggregate-translations');
+const merge = require('webpack-merge');
+
+const defaultWebpackConfig = (env = {}, argv = {}) => {
+  const production = argv.p;
+  const disableAggregateTranslations = env.disableAggregateTranslations;
+  const processPath = process.cwd();
+  /* Get the root path of a mono-repo process call */
+  const rootPath = processPath.includes('packages') ? processPath.split('packages')[0] : processPath;
+
+  const resolveModules = ['node_modules'];
+  if (!disableAggregateTranslations) {
+    aggregateTranslations({ baseDirectory: rootPath });
+    resolveModules.unshift(path.resolve(rootPath, 'aggregated-translations'));
+  }
+
+  const devConfig = {
+    entry: {
+      raf: 'raf/polyfill',
+      'babel-polyfill': 'babel-polyfill',
+    },
+    module: {
+      rules: [{
+        test: /\.(jsx|js)$/,
+        exclude: /node_modules(?!\/terra-dev-site\/src)/,
+        use: 'babel-loader',
+      },
+      {
+        test: /\.(scss|css)$/,
+        use: [
+          MiniCssExtractPlugin.loader,
+          {
+            loader: 'css-loader',
+            options: {
+              sourceMap: true,
+              importLoaders: 2,
+              localIdentName: '[name]__[local]___[hash:base64:5]',
+            },
+          }, {
+            loader: 'postcss-loader',
```

Comment:
Lets add a comment here:

```
// Add unique ident to prevent the loader from searching for a postcss.config file. Additionally see: https://github.com/postcss/postcss-loader#plugins
```